docs/appnotes/0017-reuse-of-ids.md

```diff
@@ -0,0 +1,128 @@
+They should have the same Source ID but different Flow IDs.
+
+If two Flows are technically identical but are timeshifted, the are different Flows with different Flow IDs and different Source IDs.
+They may, however, re-use the same Objects using the Segments layer of the data model to map them to the two Flow's different timelines.
+
+If two (or more) Flows are editorially different but re-use technically identical sections (i.e. basic edits), they are different Flows with different Flow IDs and different Source IDs.
+But they may re-use some of the same Objects using the Segments layer of the data model to map them to the Flow's different timelines.
+This is termed "edit by reference".
+
+![Graphic showing an audio Source with two Flows which provide different technical representations of the Source. Each Flow has multiple Segments (i.e. independently decodable sections of media) placed against a Timeline.](./images/0001-multi-mono-essence-flows-sources-fig2.png)
+
+## Interactions With Other Systems
+
+When a TAMS system is interacting with other systems with a compatible identity model (e.g. NMOS/2110), we can expand these principles to provide best practice for those interactions.
+
+If a Flow is ingested from, or outgested to a non-TAMS system and remains technically identical with no time-shift it should be the same Flow and should maintain it's Flow and Source ID (and Object IDs, where appropriate).
+
+If a Flow is ingested from, or outgested to a non-TAMS system and remains editorially identical with no timeshift, but not technically identical, it should be the same Source and should maintain it's Source ID but have different Flow IDs.
+
+If a Flow passes from one system "A" to another system "B" and back to system "A", the Flow and its metadata should be identical at both points in system "A".
+
+It is worth noting that Objects are a TAMS concept.
+While they are somewhat analogous to Grains in NMOS, there is no requirement for them to map directly.
+The important aspect is the ability of Flows to pass through multiple systems transparently.
+
+## Real-world Implications
+
+It should also be acknowledged that the real world is messy and it might not be possible to meet these recommendations at all times.
+Consider where the same media is ingested into two separate systems simultaneously.
+Ideally, these ingestors should be configured by the same coordinating system to have the same IDs.
+Indeed, dual redundant ingest systems should work in this way.
+But there may be cases where such parallel ingests are in entirely separate systems.
+
+### Avoiding Duplication
+
+Systems may wish to present a list of existing Sources/Flows on setup in such a way that users could identify the content which is being duplicated and either re-use the existing copy (avoiding an un-needed ingest), or ingest with the existing identifiers.
```

Comment:
Nit: could one or two line breaks / paragraphs be made as this is quite a long paragraph.